### ISSUE TITLE
Nothing happened when run Tutorial 25.

### ISSUE THREAD

**Issue by gekator** [emeiri/ogldev#9]
Hello)
I compile a tutorial 25 by msys2 on the Windows10. 
It is compilig well
I use some libraries from msys2. 
However when i am execute the exe file window opening and closing, without errors
Also gdb don't show problems
Where is the problemcan be?

my compiling
g++ -g -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=16 -std=c++11 -c mesh.cpp skybox.cpp skybox_technique.cpp main.cpp  "C:\Users\GubaydullinAR\Desktop\local\C\sss\cd\ogldev-source\Common"/*.cpp -I"C:\msys64\mingw64\include" -I"C:\msys64\mingw64\include\ImageMagick-7" -I"C:\Users\GubaydullinAR\Desktop\local\C\sss\cd\ogldev-source\Common\FreetypeGL" -I"C:\msys64\mingw64\include\freetype2" -I"C:\msys64\mingw64\include\assimp" -I"C:\Users\GubaydullinAR\Desktop\local\C\sss\cd\ogldev-source\Include" -I"C:\AntTweakBar\include"

g++  -std=c++11 math_3d.o skybox_technique.o main.o mesh.o skybox.o cubemap_texture.o technique.o ogldev_basic_lighting.o ogldev_util.o pipeline.o glut_backend.o camera.o ogldev_texture.o ogldev_app.o ogldev_atb.o -o main.exe -L"C:\msys64\mingw64\lib" -L"C:\Users\GubaydullinAR\Desktop\local\C\sss\cd\ogldev-source\Lib" -L"C:\AntTweakBar\lib" -lglew32  -lfreeglut -lopengl32 -lassimp -lglu32gdb -llibMagick++-7.Q16HDRI -lAntTweakBar64 -Wl,--subsystem,windows

Thank you!

**Comment by emeiri** [comment]
The texture class is using STB image by default but this demo uses a PCX image which is not supported. You can switch to ImageMagick by commenting in USE_IMAGE_MAGICK at the top of ogldev/Include/ogldev_texture.h. I see that ImageMagick is referenced on your build and link command line so hopefully it will work.

**Comment by gekator** [comment]
> The texture class is using STB image by default but this demo uses a PCX image which is not supported. You can switch to ImageMagick by commenting in USE_IMAGE_MAGICK at the top of ogldev/Include/ogldev_texture.h. I see that ImageMagick is referenced on your build and link command line so hopefully it will work.

Thanks, i changed it. However it still happens. 
I understood reason of that.
When i execute *.exe i receive Assertion Failed.
mesh.cpp
Line 138
Expression:Face.mNumIndices == 3
But don't understand meaning

**Comment by emeiri** [comment]
The assertion makes sure that the current polygon is indeed a triangle because the code is not ready for points and lines. Try to disable it and see what happens. Are you using the original model? I'll take another look at it later.

**Comment by gekator** [comment]
> The assertion makes sure that the current polygon is indeed a triangle because the code is not ready for points and lines. Try to disable it and see what happens. Are you using the original model? I'll take another look at it later.

After some checking i found i had an incorrect path to the model. Also i had problems with loading a texture. Magick didn't load it. And i can't see errors from std::cout. I inserted some MessageBoxA() for it. Reason was the Magick need its *.exe files in C:\msys64\mingw64\bin and someyhing yet. I copied my main.exe in the \bin , and hopefully it works!
